lib/msf/util/exe.rb

```diff
@@ -237,9 +238,17 @@ def self.to_win32pe(framework, code, opts = {})
 
     # Allow the user to specify their own EXE template
     set_template_default(opts, "template_x86_windows.exe")
-
+	
+	# Finally got this working
```

Comment:
Use space for indentation please. Also some comments look like WIP please remove those are no longer needed.